### ISSUE TITLE
Problem with keras >= 2.2.0

### ISSUE THREAD

**Issue by qubvel** [qubvel/segmentation_models#8]
starting from keras 2.2.0 `keras.applications` module moved to separate repo. need to fix imports.

**Comment by Gary-Deeplearning** [comment]
@qubvel  The problem has been solved?

**Comment by qubvel** [comment]
Not yet, will try to fix it today

**Comment by Gary-Deeplearning** [comment]
@qubvel Nice, Please update the README.md. and I want to ask a question, When I ues resnet34 as backbone, It seems the .weights can not be download, the download url has been deleted?

**Comment by qubvel** [comment]
@Gary-Deeplearning check just now 'resnet34' backbone for Unet model, weights have been downloaded and it works fine with keras 2.1.5. May be the problem in keras version, will check it too.

**Comment by Gary-Deeplearning** [comment]
@qubvel yeah, It works. But when I install the segmentation_models in the GPU server, it got a strange problem: ```
Collecting segmentation_models
  Could not find a version that satisfies the requirement segmentation_models (from versions: )
No matching distribution found for segmentation_models
You are using pip version 9.0.1, however version 18.0 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```
But I can also use this to  solve the install.
```
$ git clone https://github.com/qubvel/segmentation_models.git
$ cd segmentation_models
$ git submodule update --init --recursive
```
Mat it was the problem of clound.

**Comment by qubvel** [comment]
@Gary-Deeplearning  I suppose you use python version less than 3.6

**Comment by Gary-Deeplearning** [comment]
@qubvel yeah, It's python version problem. Thanks for explaining.
